system/otherLoadTest/playlist.xml

```diff
@@ -217,7 +217,6 @@
 		<groups>
 			<group>system</group>
 		</groups>
-		<platformRequirements>^os.win</platformRequirements>
```

Comment:
Could you please remove the comment associated with the exclude too? ("<!-- Excluded from Windows platform due to: https://github.com/AdoptOpenJDK/openjdk-systemtest/issues/236 -->")

Reply:
Removed the comment

Reply:
Thanks!